AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,296 @@
+the API would return empty values for the associated properties in the dictionary. This is consistent with
+existing behavior in store information retrieval API for native applications.
+
+```js
+details = {
+  installSource: "Microsoft Store",
+  campaignId: ""
+}
+```
+
+## Attribution on browser installs
+
+While attribution is currently not being tracked in the same capacity as Store installations for browser-initiated
+installations of web applications, the Acquisition Info API could enable this sort of attribution capability.
+
+### Same-domain installation
+
+Currently the main method of installing a PWA through the browser is navigating to the desired web application and
+initiating a same-domain installation. It may be the case that the intial navigation was due to an ad click.
+By having a standardized dictionary property (`attributionId`) that the UA is responsible for capturing to keep track of ad attribution,
+it becomes possible for the Acquistion Info API to return attribution information for browser-installed
+web applications as well. This `attributionId` dictionary property could be captured by the UA through a query parameter such
+as `__a_id` in the [HTTP header `Referer`][HTTP Referer] field.
+> This solution would require that the referrer policy is not specified to block `referer` information requests.
+
+#### Example same-domain use case
+
+1. User clicks on a Bing ads campaign on `bar.com` that navigates to `foo.com`.
+The HTTP `Referer` passed to `foo.com` is `bar.com?__a_id=bingAdsAug2023`.
+
+2. User installs the `foo.com` PWA from `foo.com`.
+
+3. UA captures the attribution information at the time of installation.
+
+4. Calling the Acquisition Info API `getDetails()` for `foo.com` PWA produces the following payload.
+
+> Proposed dictionary properties are not final and subject to change based on feedback.
+
+```js
+details = {
+  installSource: "foo.com",
+  attributionId: "bingAdsAug2023",
+  httpReferrer: "bar.com?__a_id=bingAdsAug2023",
+  installTimestamp: "2023-08-16 10:30:00 UTC"
+}
+```
+
+### Web Install API (Cross-domain installation)
+
+With the addition of the [Web Install API][Web Install API], there exists a possibility of cross-domain
+browser-initiated installations that may have associated attribution information through the new capability
+of webapps to install other webapps. The Web Install API contains a `referral-info` parameter for its installation
+method, which contains an object that can hold arbitrary attribution information. The UA can capture the information
+contained in `referral-info` at install time so that the Acquisition Info API can return the data later.
+
+#### Example cross-domain use case
+
+1. User clicks on a Bing ads campaign on `bar.com` that navigates to the Microsoft Store website product landing
+page for `foo.com`.
+
+2. The query param `__a_id=bingAdsAug2023` is inserted into the HTTP `Referer` passed to the Microsoft Store website on navigation,
```

Comment:
It's not inserted into the HTTP Referer.  It's inserted into the URL being navigated _to_, e.g. `apps.microsoft.com?__a_id=bingAdsAug2023`